### ISSUE TITLE
400. Error: redirect_uri_mismatch

### ISSUE THREAD

**Issue by miyconst** [GoogleWebComponents/google-signin#76]
`<google-signin>` stopped working.

The demo at:
http://googlewebcomponents.github.io/google-signin/components/google-signin/demo.html

Returns 400. Error: `redirect_uri_mismatch` for every button.

![google-signin-error](https://cloud.githubusercontent.com/assets/6435556/6826363/feabde38-d311-11e4-9ab2-ddac3959c77d.PNG)

**Comment by Zoramite** [comment]
Thanks for pointing this out. I'll check into it and see if this is an expected change with the updated API or not.

**Comment by Zoramite** [comment]
@miyconst We found the issue. The developer console for the project was setup for `http://googlewebcomponents.github.io/` but for some reason the origin only accepts the `http://googlewebcomponents.github.io` format. It has been updated and should be working now.

We are looking into why it does not work when the domain has a trailing slash, but for now removing the trailing slash should work if you run into this on other domains.
